templates/tapestry.js

```diff
@@ -637,72 +639,6 @@ function tapestryTool(config){
             });
     }
 
-    // D3 DRAGGING FUNCTIONS
```

Comment:
Can you please move this back up here? It shouldn't be down there as it is out of place. Also hard to review the changes like this.

Reply:
I understand the difficulty in reviewing changes, but I feel it's better suited down below since it's only being used in one place only, i.e. where we're setting up node dragging. I feel it's more difficult having to jump to different places in the file when working on the drag functionality.